Dockerfile

```diff
@@ -19,18 +19,26 @@ RUN apt-get update && \
 RUN apt-get update && \
     apt-get install -y ffmpeg
 
+RUN apt-get install -y curl
+
 # Copy the published application
 COPY . /source
 
+RUN UUID=$(cat /proc/sys/kernel/random/uuid) && \
+    FFMPEG_PATH=$(which ffmpeg) && \
+    FFPROBE_PATH=$(which ffprobe) && \
+    echo '{ "BunnyStorageZoneName": "", "BunnyCdnRootUrl": "", "BunnyAccessKey": "", "BunnyBasePath": "", "BunnyTokenKey": "","BunnyUserApiKey": "", "AuthSignaturePrivateKey": "'$UUID'", "FfprobePath": "'$FFPROBE_PATH'", "FfmpegPath": "'$FFMPEG_PATH'"}' > /source/config/settings.json
```

Comment:
Next step might be to let docker ask for the bunny info (URL and keys) in prompts.